ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Thanks for using Certbot - if you're experiencing issues with using Certbot
```

Comment:
I personally don't think any intro is necessary and we can just start with questions. It makes it less to read and is the format I've seen on most other repos. Feel free to leave it if you disagree though.